src/main/java/io/seqera/tower/cli/commands/data/links/SearchOption.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2021-2023, Seqera.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package io.seqera.tower.cli.commands.data.links;
+
+import io.seqera.tower.cli.utils.data.DataLinkProvider;
+import picocli.CommandLine;
+
+public class SearchOption {
+
+    @CommandLine.Option(names = {"-n", "--name"}, description = "Show only data links with names that start with the given word.")
+    public String startsWith;
+
+    @CommandLine.Option(names = {"-r", "--region"}, description = "Show only data links belonging to given region")
+    public String region;
+
+    @CommandLine.Option(names = {"-p", "--providers"}, description = "Show only data links belonging to given providers. [aws, azure, google]")
```

Comment:
is the providers option expected to be comma separated?

Reply:
Yes, we use standard methods to process search query:
https://github.com/seqeralabs/platform/blob/962e0a1b13ab120fed8faaff7b4fdd513ffb50b0/tower-services/src/main/groovy/io/seqera/tower/util/search/SearchQuery.groovy#L198